metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -533,6 +533,17 @@ object MetalsEnrichments
       filename.endsWith(".jar") || filename.endsWith(".srcjar")
     }
 
+    def toTargetRoot: AbsolutePath = {
+      if (path.isJar) {
+        val filename = path.toNIO.getFileName.toString
+        val filename0 = filename.stripSuffix(".jar").stripSuffix(".srcjar")
+        val targetroot = path.parent.resolve(s"_semanticdb/$filename0")
```

Comment:
How does the directory structure look like in case of Bazel? `_semanticdb` seems like it could be fragile.

Reply:
Based on simple projects I've seen, the `_semanticdb` directory gets created in the same place as the targetroot jar.

Reply:
Let's add a comment that it's used for Bazel.